project.yaml

```diff
@@ -117,6 +117,25 @@ targets:
           exhaustive:
           - rhel-cloud:rhel-9-arm64
           - rocky-linux-cloud:rocky-linux-9-optimized-gcp-arm64
+  rockylinux10:
+    os_versions: [rocky-10*,rhel-10*]
```

Comment:
Just to confirm, Is adding `centos-10` a separate task ?

Reply:
Yes, centos-stream-10 is a separate task